main.js

```diff
@@ -133,18 +137,21 @@ function disableSaveButton(event){
 
 function saveOutfit() {
   var outfitName = saveOutfitInput.value;
-  var savedOutfitCard = `<button class="button-style">${outfitName}<img class="close-btn" src="assets/close.svg" alt="Close"></button>`;
+  var outfitId = newOutfit.id;
+  var savedOutfitCard = `<button class="button-style">${outfitName}<img id="${outfitId}" class="close-btn" src="assets/close.svg" alt="Close"></button>`;
   newOutfit.title = outfitName;
   savedOutfitsList.insertAdjacentHTML('beforeend', savedOutfitCard);
-  window.localStorage.setItem(outfitName, savedOutfitCard);
+  window.localStorage.setItem(outfitId, JSON.stringify(newOutfit));
```

Comment:
Saving outfit object to localstorage instead of just html. Had to convert it to a string in order to save to localStorage.